public/theme.css

```diff
@@ -0,0 +1,83 @@
+/*!
```

Comment:
Excellent choice to use Bootstrap for your design! We'll be learning more about this when we move onto Rails.